MPF.md

```diff
@@ -974,6 +974,40 @@ For example:
 
 **History**: Added in 3.10.0
 
+### Configure `abortclasses` via augments
+
+Configure a list of regular expressions that result in cf-agent terminating
+itself upon definition of a matching class.
+
+For example, abort execution if any class starting with ```error_``` or ```abort_``` is defined:
+
+```
+{
+  "vars":{
+    "control_agent_abortclasses": [ "error.*", "abort.**" ]
+  }
+}
+```
+
+**History**: Added in 3.15.0, 3.12.3
```

Comment:
why is 3.15.0 included here?

Reply:
It's the next version that this will be part of (after 3.14)

Reply:
Oh you mean in this branch of docs? I haven't ever made sure that old branches do not mention newer ones. I can yank it out though if you think i should.